deployment/cdk/opensearch-service-migration/default-values.json

```diff
@@ -4,5 +4,6 @@
   "migrationAssistanceEnabled": true,
   "migrationConsoleServiceEnabled": true,
   "trafficReplayerServiceEnabled": false,
-  "otelCollectorEnabled": true
+  "otelCollectorEnabled": true,
+  "reindexFromSnapshotMaxShardSizeGiB": 80
```

Comment:
Can we move this to cdk.context.json? I'd like to increase visibility into this value

Reply:
Oh, good catch, +1